reorganize.py

```diff
@@ -0,0 +1,169 @@
+# reorganize.py
+# this program takes the output of the outbreak simulator and converts it into a csv file with relevant data shown in tabular format 
+# average values are given first, and individual results for each parameter for each simulation are given
+
+#imports needed libraries
+import numpy as np
+import pandas as pd
+import re
+import matplotlib.pyplot as plt
+import argparse
+
+def reorganize(file):
+    def createDictionary(txtfile): #converts txt file to dictionary
+        with open(txtfile) as f:
+            pq_lines = f.readlines()
+
+        for i in range(0, len(pq_lines)):
+            split = pq_lines[i].split("\t")
+            key = split[0]
+            value = split[1]
+            pq_dict[key] = value
+        print('Dictionary Created')
+
+        # delete keys that can't be turned into table form

```

Comment:
Cannot you judge this by the lack of time pattern? Basically when you get a bunch of keys, some of them would be `A_0.00`, `A_1.00` etc, some woudl be `C`, `D`, so you can do something like

1. separate all keys by name and time stamp. You would get things like `['', '', '', '1.0', '2.0', '1.0', '2.0']` for the time stamp portion. 

2. Identify the `[`1.0', '2.0']` stuff from the list.

3. Remove all keys without timestamp or does not full fill this time pattern. 

Very quickly I can think of something like:

```
separated = [x.rsplit('_', 1) for x in keys if '_' in x]
times = set([x[1] for x in separated if x.replace('.', '').isnumeric()])
...
```